onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -532,6 +532,7 @@ def _get_quantize_input_nodes(self, node, input_index, qType, given_scale_name=N
                 return None
             # dynamic mode
             # Scale and Zero Points not available for this input. Add nodes to dynamically compute it
+            # TODO: Support DynamicQuantizeLinear for s8 after onnxruntime have it.
```

Comment:
>DynamicQuantizeLinear [](http://example.com/codeflow?start=28&length=21)

DynamicQuantizeLinear only supports uint8

Reply:
Yeah, currently it is, and it is hard to decide type from output type?
But It is finally will be asked if S8S8 dynamic is used, we may need other solutions.